paimon-web-server/src/test/java/org/apache/paimon/web/server/controller/TableControllerTest.java

```diff
@@ -172,15 +172,17 @@ public void testDropColumn() throws Exception {
     @Test
     public void testAlterTable() throws Exception {
         TableColumn oldColumn =
-                new TableColumn("id", PaimonDataType.builder().type("INT").build(), "", false, "0");
+                new TableColumn(
+                        "name", PaimonDataType.builder().type("STRING").build(), "", false, "0");
 
         TableColumn newColumn =
                 new TableColumn(
-                        "age", PaimonDataType.builder().type("BIGINT").build(), "", false, "0");
+                        "age", PaimonDataType.builder().type("BIGINT").build(), "", false, "1");
 
         AlterTableRequest alterTableRequest = new AlterTableRequest();
         alterTableRequest.setOldColumn(oldColumn);
         alterTableRequest.setNewColumn(newColumn);
+        alterTableRequest.setMoveToFirst(true);
```

Comment:
Verify whether `moveToFirst` and `afterColumName` is correct.

Reply:
OK, thank you for your review. After the PR is merged at https://github.com/apache/incubator-paimon-webui/pull/146, I will add test to verify whether moveToFirst and afterColumName is correct.